framework/src/outputs/TableOutput.C

```diff
@@ -34,6 +34,9 @@ InputParameters validParams<TableOutput>()
   InputParameters params = validParams<AdvancedOutput<FileOutput> >();
   params += AdvancedOutput<FileOutput>::enableOutputTypes("postprocessor scalar vector_postprocessor");
 
+  // Option for writing vector_postprocessor time file
+  params.addParam<bool>("time_data", false, "When true and VecptorPostprocessor data exists, write a csv file containing the timestep and time information.");
```

Comment:
Thanks, at some point in the near future our continued evolution of the SQA process is going to require us to maintain a "design" document. This is the kind of information that needs to appear in that document. I also know that if that document is even a little bit static, it'll be out of date immediately. We have started the discussion here about how to evolve our doxygen markings even more so that such a document could be largely built dynamically. Yes, even that's not perfect unless we require design descriptions in each patch. Lots to figure out...